cmd/uninstall.go

```diff
@@ -70,7 +71,17 @@ func deleteSourceDirs(fs afero.Fs) {
 // DeleteGitHooks read the config and remove all git hooks except
 func DeleteGitHooks(fs afero.Fs) {
 	hooksPath := filepath.Join(getRootPath(), ".git", "hooks")
-	hooks, _ := afero.ReadDir(fs, hooksPath)
+
+	hooks, err := afero.ReadDir(fs, hooksPath)
+	if (err != nil) {
+		log.Println("⚠️ ", au.Bold(".git/hooks"), "directory does not exist, creating")
+
+		if err := os.MkdirAll(hooksPath, os.ModePerm); err != nil {
```

Comment:
I hope we will change it back to `os.Mkdir` while fixing https://github.com/evilmartians/lefthook/issues/90